cmd/kubeapps-apis/plugins/fluxv2/packages/v1alpha1/oci_repo.go

```diff
@@ -380,23 +384,17 @@ func newRegistryClient(isLogin bool, tlsConfig *tls.Config, getterOpts []getter.
 // OCI Helm repository, which defines a source, does not produce an Artifact
 // ref https://fluxcd.io/docs/components/source/helmrepositories/#helm-oci-repository
 
-// TODO: this function is way too long. Break it up
 func (s *repoEventSink) onAddOciRepo(repo sourcev1.HelmRepository) ([]byte, bool, error) {
 	log.Infof("+onAddOciRepo(%s)", common.PrettyPrint(repo))
 	defer log.Info("-onAddOciRepo")
 
+	// I am disabling repo cache due to
```

Comment:
Are you or is it just a leftover comment? It seems you are still using the `cacheEntryValue`, no?

Reply:
just left over comment. WIll fix